### PR TITLE
docstrings: `docstore` and `client`

### PR DESCRIPTION
**Pull request by leo-gan**
updated docstrings in `docstore/` and `client/`

@baskaryan

### CODE REVIEW

**Comment by leo-gan** [comment]
@hwchase17 Can you review it, please? TNX!

**Comment by leo-gan** [comment]
@rlancemartin, @eyurtsev can you, please, review it? Thanks!

**Comment by leo-gan** [comment]
@rlancemartin  Please review it. Thanks!

**Comment by leo-gan** [comment]
@baskaryan Please review it. Thanks!

File: langchain/client/runner_utils.py [hunk 1]
```diff
@@ -342,8 +342,8 @@ async def arun_on_examples(
         llm_or_chain_factory: Language model or Chain constructor to run
             over the dataset. The Chain constructor is used to permit
             independent calls on each example without carrying over state.
-        concurrency_level: The number of async tasks to run concurrently.
-        num_repetitions: Number of times to run the model on each example.
+        concurrency_level: [Default: 5] The number of async tasks to run concurrently.
```
Comment (baskaryan) [inline langchain/client/runner_utils.py:345]: no need to note defaults in docstrings i think, that's already made clear in function type hints

File: langchain/docstore/arbitrary_fn.py [hunk 2]
```diff
@@ -21,6 +21,14 @@ def __init__(
         self._lookup_fn = lookup_fn
 
     def search(self, search: str) -> Document:
+        """
+        Search for a document.
```
Comment (baskaryan) [inline langchain/docstore/arbitrary_fn.py:25]: nit: move this up a line, leave break b/n first line and Args:

File: langchain/client/runner_utils.py (hunk 1 above)
Comment (leo-gan) [reply langchain/client/runner_utils.py:345]: - [+]

File: langchain/docstore/arbitrary_fn.py (hunk 2 above)
Comment (leo-gan) [reply langchain/docstore/arbitrary_fn.py:25]: - [+]

**Comment by baskaryan** [comment]
thanks @leo-gan!

**Pull request merged**